<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>9</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClean"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionCircle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionRect"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionCurve"/>
   <addaction name="actionFill"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="separator"/>
   <addaction name="actionLarge"/>
   <addaction name="actionLittle"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/直线.bmp</normaloff>:/icons/直线.bmp</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="statusTip">
    <string>画一条直线</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/圆.bmp</normaloff>:/icons/圆.bmp</iconset>
   </property>
   <property name="text">
    <string>圆</string>
   </property>
   <property name="statusTip">
    <string>画一个圆</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/椭圆.bmp</normaloff>:/icons/椭圆.bmp</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
   <property name="statusTip">
    <string>画一个椭圆</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/矩形.bmp</normaloff>:/icons/矩形.bmp</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="statusTip">
    <string>画一个矩形</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/多边形.bmp</normaloff>:/icons/多边形.bmp</iconset>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
   <property name="statusTip">
    <string>画一个多边形。单击鼠标左键并拖动画线段，单击右键结束画图</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/曲线.bmp</normaloff>:/icons/曲线.bmp</iconset>
   </property>
   <property name="text">
    <string>曲线</string>
   </property>
   <property name="statusTip">
    <string>贝塞尔曲线。单击鼠标左键并拖动画辅助线，当画完三条辅助线后生成曲线</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/颜料桶.bmp</normaloff>:/icons/颜料桶.bmp</iconset>
   </property>
   <property name="text">
    <string>填充</string>
   </property>
   <property name="statusTip">
    <string>默认为黑色，修改请点击颜色设置。</string>
   </property>
  </action>
  <action name="actionLarge">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/放大.bmp</normaloff>:/icons/放大.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="statusTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionLittle">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/缩小.bmp</normaloff>:/icons/缩小.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="statusTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/颜色.bmp</normaloff>:/icons/颜色.bmp</iconset>
   </property>
   <property name="text">
    <string>颜色设置</string>
   </property>
   <property name="statusTip">
    <string>设置颜色，填充模式解锁，只能修改填充颜色，不能修改图形边框颜色。</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/矩形选择框.bmp</normaloff>:/icons/矩形选择框.bmp</iconset>
   </property>
   <property name="text">
    <string>选择框</string>
   </property>
   <property name="statusTip">
    <string>选择要裁剪的直线范围</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/裁剪.bmp</normaloff>:/icons/裁剪.bmp</iconset>
   </property>
   <property name="text">
    <string>裁剪</string>
   </property>
   <property name="statusTip">
    <string>先画选择框再裁剪。选择框模式解锁。</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/保存.bmp</normaloff>:/icons/保存.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/打开.bmp</normaloff>:/icons/打开.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开3D图像</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/新建.bmp</normaloff>:/icons/新建.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空当前画布</string>
   </property>
   <property name="statusTip">
    <string>清空当前画布</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/提示.bmp</normaloff>:/icons/提示.bmp</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>查看帮助</string>
   </property>
   <property name="statusTip">
    <string>查看帮助</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
